blueocean-bitbucket-pipeline/src/test/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/cloud/BitbucketApiTest.java

```diff
@@ -32,7 +32,7 @@
  */
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({Secret.class})
-@PowerMockIgnore({"javax.crypto.*", "javax.security.*", "javax.net.ssl.*", "javax.net.SocketFactory"})
+@PowerMockIgnore({"javax.crypto.*", "javax.security.*", "javax.net.ssl.*", "com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*", "org.w3c.dom.*"})
```

Comment:
I didn't know thats a thing. looks like https://github.com/powermock/powermock-examples-maven/blob/7b57a10b27e174ff99ee3892ea402da8275a8610/global-ignore/src/test/resources/org/powermock/extensions/configuration.properties is an example to me.

Do you know how to get that working in multi module? do I create the src/test/resources directory on the parent?

Reply:
@kzantow so as we learned "offline", its not super simple to do. Best suggestion is another maven plugin that copies a global config everywhere.

I honestly don't see a problem with the copy and paste, especially for tests, especially since they break without it.

Reply:
I definitely agree. Only hold up is trying to get a successful ATH run, imo